wordpress-mobile/WordPress-iOS
# Remove custom activities

Closes #4600 

How to test:

Go to the reader, open an article and click on the share extension:

- The following custom activities should be gone:

- GooglePlus
- Instapaper
- Pocket

However if you have any of those applications installed in your device you should be to use their standard share activity that should be available on the top row, if you have enable it.

@astralbodies: I noticed that we where tracking the use of these custom activities, so I kept the tracking on using the extension class identifier. Is this something we want to keep? Or we just want to track the general use of share activities and those are not that relevant?

Needs Review: @aerych
